pyproject.toml

```diff
@@ -1,6 +1,7 @@
 [tool.poetry]
 name = "pipeline-ai"
-version = "0.0.49"
+version = "1.0.0"
```

Comment:
sorry, my bad, @alexpearce pointed out that if we're not yet on version 1 we can still introduce breaking changes

Reply:
Okay